note on building in a 27 conda env. 
- install scipy
- in **bash** use `ARCHIVE=1 VERSION=3 make gh-page` to build

investigate deeper : 
- try `conda install -c r r` to get R
- then `pip install rpy2`

rpy2 docs still not building. 

Seem linked to DYLD_LIb_PATH found this thread but does not work: 
https://groups.google.com/a/continuum.io/forum/#!topic/anaconda/EKoqtSbwhts

```
In [1]: load_ext IPython.extensions.rmagic
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-96b4c0e76842> in <module>()
----> 1 get_ipython().magic(u'load_ext IPython.extensions.rmagic')

/Users/bussonniermatthias/dev/ipython/IPython/core/interactiveshell.pyc in magic(self, arg_s)
   2163         magic_name, _, magic_arg_s = arg_s.partition(' ')
   2164         magic_name = magic_name.lstrip(prefilter.ESC_MAGIC)
-> 2165         return self.run_line_magic(magic_name, magic_arg_s)
   2166
   2167     #-------------------------------------------------------------------------

/Users/bussonniermatthias/dev/ipython/IPython/core/interactiveshell.pyc in run_line_magic(self, magic_name, line)
   2084                 kwargs['local_ns'] = sys._getframe(stack_depth).f_locals
   2085             with self.builtin_trap:
-> 2086                 result = fn(*args,**kwargs)
   2087             return result
   2088

/Users/bussonniermatthias/dev/ipython/IPython/core/magics/extension.pyc in load_ext(self, module_str)

/Users/bussonniermatthias/dev/ipython/IPython/core/magic.pyc in <lambda>(f, *a, **k)
    189     # but it's overkill for just that one bit of state.
    190     def magic_deco(arg):
--> 191         call = lambda f, *a, **k: f(*a, **k)
    192
    193         if callable(arg):

/Users/bussonniermatthias/dev/ipython/IPython/core/magics/extension.pyc in load_ext(self, module_str)
     60         if not module_str:
     61             raise UsageError('Missing module name.')
---> 62         res = self.shell.extension_manager.load_extension(module_str)
     63
     64         if res == 'already loaded':

/Users/bussonniermatthias/dev/ipython/IPython/core/extensions.pyc in load_extension(self, module_str)
     96             if module_str not in sys.modules:
     97                 with prepended_to_syspath(self.ipython_extension_dir):
---> 98                     __import__(module_str)
     99             mod = sys.modules[module_str]
    100             if self._call_load_ipython_extension(mod):

/Users/bussonniermatthias/dev/ipython/IPython/extensions/rmagic.py in <module>()
     53 import numpy as np
     54
---> 55 import rpy2.rinterface as ri
     56 import rpy2.robjects as ro
     57 try:

/Users/bussonniermatthias/anaconda3/envs/27/lib/python2.7/site-packages/rpy2/rinterface/__init__.py in <module>()
     98
     99
--> 100 from rpy2.rinterface._rinterface import *
    101
    102

ImportError: dlopen(/Users/bussonniermatthias/anaconda3/envs/27/lib/python2.7/site-packages/rpy2/rinterface/_rinterface.so, 2): Library not loaded: @rpath/R/lib/libR.dylib
  Referenced from: /Users/bussonniermatthias/anaconda3/envs/27/lib/python2.7/site-packages/rpy2/rinterface/_rinterface.so
  Reason: image not found
```

Investigate the following: 

```
/Users/bussonniermatthias/dev/ipython/docs/source/api/generated/IPython.qt.rich_text.rst:31: WARNING: autodoc: failed to import function u'export_xhtml' from module u'IPython.qt.rich_text'; the following exception was raised:
Traceback (most recent call last):
  File "/Users/bussonniermatthias/anaconda3/envs/27/lib/python2.7/site-packages/Sphinx-1.2.3-py2.7.egg/sphinx/ext/autodoc.py", line 335, in import_object
    __import__(self.modname)
  File "/Users/bussonniermatthias/dev/ipython/IPython/qt/rich_text.py", line 13, in <module>
    from IPython.external.qt import QtGui
  File "/Users/bussonniermatthias/dev/ipython/IPython/external/qt.py", line 23, in <module>
    QtCore, QtGui, QtSvg, QT_API = load_qt(api_opts)
  File "/Users/bussonniermatthias/dev/ipython/IPython/external/qt_loaders.py", line 258, in load_qt
    api_options))
ImportError:
    Could not load requested Qt binding. Please ensure that
    PyQt4 >= 4.7 or PySide >= 1.0.3 is available,
    and only one is imported per session.

    Currently-imported Qt library:   'pyqtv1'
    PyQt4 installed:                 True
    PySide >= 1.0.3 installed:       False
    Tried to load:                   ['pyside', 'pyqt']
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -11,7 +11,7 @@ before_install:
     - easy_install -q pyzmq
     - sudo apt-get install pandoc
     - pip install 'pygments<1.5'
-    - pip install sphinx
+    - pip install 'sphinx<1.3'
 install:
     - python setup.py install -q 
 script:

--- a/.travis.yml
+++ b/.travis.yml
@@ -10,7 +10,7 @@ before_install:
     - "if [[ ! $TRAVIS_PYTHON_VERSION == '3.2'* ]]; then pip install jinja2; fi"
     - easy_install -q pyzmq
     - sudo apt-get install pandoc
-    - pip install pygments
+    - pip install 'pygments<1.5'
     - pip install sphinx
 install:
     - python setup.py install -q 

--- a/setup.py
+++ b/setup.py
@@ -273,12 +273,12 @@ if 'setuptools' in sys.modules:
     setuptools_extra_args['entry_points'] = find_scripts(True)
     setup_args['extras_require'] = dict(
         parallel = 'pyzmq>=2.1.11',
-        qtconsole = ['pyzmq>=2.1.11', 'pygments'],
+        qtconsole = ['pyzmq>=2.1.11', 'pygments<1.5'],
         zmq = 'pyzmq>=2.1.11',
         doc = 'Sphinx>=0.3',
         test = 'nose>=0.10.1',
         notebook = ['tornado>=2.0', 'pyzmq>=2.1.11', 'jinja2'],
-        nbconvert = ['pygments', 'jinja2', 'Sphinx>=0.3']
+        nbconvert = ['pygments<1.5', 'jinja2', 'Sphinx>=0.3']
     )
     everything = set()
     for deps in setup_args['extras_require'].values():

